node/resolvers/checkout/orderFormItem.ts

```diff
@@ -0,0 +1,19 @@
+import { AssemblyOption } from './types'
+
+interface Params extends OrderFormItem{
+  assemblyOptionsData: {
+    childs: OrderFormItem[]
+    index: number
+    assemblyOptionsMap: Record<string, AssemblyOption[]>
+    orderForm: any
+  }
+}
+
+export const resolvers = {
+  OrderFormItem: {
+    assemblyOptions: ({ assemblyOptionsData: { childs, index, orderForm, assemblyOptionsMap }, ...item }: Params) => ({ item, childs, index, orderForm, assemblyOptionsMap }),
+    listPrice: ({ listPrice }: Params) => listPrice / 100,
```

Comment:
extract this magic number to a const

Reply:
this is not quite a magic number, it's just changing the decimal place. in this case I think it's clearer like this than having another variable

Reply:
I think that's just how-the-API-works-with-prices-magic-number. I don't think it's needed.

Reply:
Yeah. Checkout returns numbers as integers. Like, R$100,00 (a hundred reais), will be represented as `10000`, that is why we usually divide by 100. It already happened before, I just moved this resolvers here, so should not break anything.